docs/_source/tutorials-examples/population-synthesis/bbh_analysis.ipynb

```diff
@@ -183,20 +150,13 @@
    "outputs": [],
    "source": [
     "from posydon.popsyn.synthetic_population import Population\n",
+    "from posydon.config import PATH_TO_POSYDON_DATA\n",
     "\n",
-    "PATH_TO_POSYDON_DATA = '/Users/max/Documents/POSYDON_data/240305/POSYDON_data/tutorials/population-synthesis/example/'\n",
-    "\n",
-    "files = ['1e-01_Zsun_population.h5',\n",
-    "         '1e-02_Zsun_population.h5',\n",
-    "         '1e-03_Zsun_population.h5',\n",
-    "         '1e-04_Zsun_population.h5',\n",
-    "         '1e+00_Zsun_population.h5',\n",
-    "         '2e-01_Zsun_population.h5',\n",
-    "         '2e+00_Zsun_population.h5',\n",
-    "         '4.5e-01_Zsun_population.h5']\n",
     "\n",
+    "data_path = f'PATH/TO/TUTORIAL/DATA/'\n",
     "\n",
-    "pop = Population(PATH_TO_POSYDON_DATA+files[0])"
+    "# Let's load one of the populations\n",
+    "pop = Population(data_path+'1e-01_Zsun_population.h5')"
```

Comment:
You added the import of `PATH_TO_POSYDON_DATA` and removed the usage of it. Shouldn't you just do either of it?